app/src/main/java/fr/publicissapient/planningpoker/model/Card.kt

```diff
@@ -0,0 +1,9 @@
+package fr.publicissapient.planningpoker.model
+
+import androidx.compose.ui.text.AnnotatedString
+
+data class Card(
+	val imageResourceId: Int,
```

Comment:
Ajouter `@DrawableRes` ?

Reply:
Yes, you may do it if I don't do it first :)